source/DSCResources/RemoteDesktopDeployment/RemoteDesktopDeployment.schema.psm1

```
@@ -6,11 +6,11 @@ configuration RemoteDesktopDeployment
         [string]
         $ConnectionBroker,
 
-        [Parameter()]
+        [Parameter(Mandatory = $true)]
```

PSScriptAnalyzer (1.24.0)

[error] Unexpected attribute 'Parameter'.

(UnexpectedAttribute)